doc/09-object-types.md

```diff
@@ -1391,9 +1391,9 @@ Configuration Attributes:
   environment               | String                | **Optional.** Specify the Icinga environment. This overrides the `Environment` constant specified in the configuration or on the CLI with `--define`. Defaults to empty.
 
 
-### IcingaDB <a id="objecttype-icingadb"></a>
+### Icinga DB <a id="icinga-db"></a>
```

Comment:
This would make this the only section that's not named exactly like the object type.